src/services/Libs.ts

```diff
@@ -344,12 +344,15 @@ export const parseCookieStoreId = (
 /**
  * Prepare Domains for all cleanups.
  */
-export const prepareCleanupDomains = (domain: string): string[] => {
+export const prepareCleanupDomains = (
+  domain: string,
+  isChrome: boolean,
```

Comment:
Same situation as before. Browser enum and switch would be more extendable.

Reply:
Implemented in commit ac90490